apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -109,10 +109,17 @@ export const gradeFormatter = function (grade, {rowData}) {
 
 export const loginInfoFormatter = function (loginType, {rowData}) {
   let sectionCode = '';
-  if (rowData.providerManaged) {
+  let pegasusUrl = pegasus('/teacher-dashboard#/sections/' + rowData.id + '/print_signin_cards');
+  if (rowData.providerManaged){
+    let providerName;
+    if (rowData.loginType === OAuthSectionTypes.clever){
+      providerName = i18n.loginTypeClever();
+    } if (rowData.loginType === OAuthSectionTypes.google_classroom) {
+      providerName = i18n.loginTypeGoogleClassroom();
+    }
     sectionCode = (
-      <div data-tip={i18n.providerManagedSection({provider: rowData.loginType})}>
-        {i18n.none()}
+      <div data-tip={i18n.providerManagedSection({provider: providerName})}>
+        {providerName}
```

Comment:
It's hard to tell without additional background knowledge here, but are we pretty sure `providerName` will never be undefined here? It wouldn't be too bad if "" (or "undefined") appeared in this column, but the tooltip message would be pretty confusing.

Reply:
Sections should only be provider managed if they are GC or Clever, so I think this is safe.